### DISCUSSION TITLE
Newbie (possible) stupid question

### DISCUSSION THREAD

**Discussion by marcio-jr** [community/community#22353] [New to GitHub]
<p>Hi everybody, I am sorry if this was already answered and I just didn’t found the topic and I am also sorry if it sounds too stupid for you guys.</p>
<p>Anyway, my question is: Is it possible to clone a repository from a Github to another Github without using any local tool?</p>
<p>The goal is to be able to create a new Github containing a base project(which is at another github), even when not able to instal the Git locally. The Github website interface is VERY usefull and one can do almost everything using a regular folder on the computher (or a pendrive) without having Git on it. But to start over, it’s pointless to copy  some 100s of files manually.</p>
<p>Thanks a lot,</p>
<p>Marcio Jr.</p>

**Comment by djkeh** [comment, marked as answer]
<p>If you mean copying a remote project to your remote repository, yes you have a solution: <code>Github fork</code></p>
<p>If you mean copying a remote project to your local computer, I think <code>git clone</code> is the clean and only answer.</p>
<p>However, if you still simply want a copy to your local place and you don’t really care about any ideas of version control, so that’s the reason you didn’t want to install git, well Github can do that for you: <code>Download ZIP</code>.</p>
<ol>
<li>find this from Github project main page.</li>
</ol>
<p><img src="https://user-images.githubusercontent.com/4949544/181070301-88012204-2efa-4a75-92ed-a4c7ba4c1c95.png" alt="image.png" data-base62-sha1="5a4GBkdtoDNIDxOoF4Gvtz4gk3W" width="386" height="281"></p>
<p>You’ll instantly get a zip file.</p>
<ol start="2">
<li>Get a link of it</li>
</ol>
<p>That <code>download zip</code> link has a certain pattern if the owner didn’t customize the project option.</p>
<pre><code>https://github.com/{account}/{project}/archive/{branch-name}.zip
</code></pre>
<p><code>branch-name</code> usually goes “master” as default project option.</p>
<p>Once you get the link, you do <code>curl</code> or something.</p>

**Comment by marcio-jr** [comment]
<p>Thanks a lot, djkeh!</p>
<p>I really din’t know that I could fork this way! That’s just great!</p>
<p>Very wise that you also explain the clone and download option, since other people might be interested when see my question.</p>
<p>Congrats.</p>
<p>Bye,</p>
<p>Marcio Jr.</p>

**Comment by missmatsuko** [comment]
<p><a class="mention" href="/djkeh">@djkeh</a> I am wondering if that URL format is documented somewhere?</p>
<p>I saw from this Stack Overflow post that the branch name part could also be a commit hash: &lt;<a href="https://stackoverflow.com/a/32340381/6387394" rel="nofollow noopener">https://stackoverflow.com/a/32340381/6387394</a>&gt;.</p>
<p>I’m exploring the feasability of combining several repos into one, with a separate branch for each repo. It’s essentially a slide deck and each branch would have different content. But the audience would have to be able to download the slide deck (a branch) for local use without using a command line, since they’re beginner tech students.</p>
